wdio-test-runner/typescript/basic/package.json

```diff
@@ -15,7 +15,7 @@
     "@wdio/mocha-framework": "^8.11.0",
     "@wdio/spec-reporter": "^8.11.2",
     "chai": "^4.3.7",
-    "chromedriver": "^132.0.0",
+    "chromedriver": "^134.0.0",
```

Comment:
NBD - small enough PR where this doesn't matter too much but IMO these should be separate PR's